charts/cactus-parser/values.yaml

```
@@ -0,0 +1,316 @@
+# Default values for cactus-parser.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: 992382468184.dkr.ecr.eu-central-1.amazonaws.com/cactus-pidb
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "feature-integration-api-cleanup"
```

Needs to be set back to empty tag so it uses the chart `appVersion` before merging.
```suggestion
  tag: ""
```
Or maybe it needs to be `latest` if that's what we recommend people use. In that case the comment should also be removed.

---

updated